### PR TITLE
info page

### PR DESCRIPTION
**Pull request by saihaj**
Just to have something out there before we bring new things.

### CODE REVIEW

File: src/pages/_document.tsx [hunk 1]
```diff
@@ -46,6 +46,29 @@ class CustomDocument extends Document {
...
+          <meta name="twitter:card" content="summary_large_image" />
+          <meta property="twitter:url" content="https://scribedao.com/" />
+          <meta name="twitter:title" content="ScribeDAO" />
+          <meta
+            name="twitter:description"
+            content="Curating and simplifying the endless stream of crypto content."
+          />
+          <meta
+            name="twitter:image"
+            content="https://scribedao.vercel.app/static/favicons/apple-touch-icon-120x120.png"
```
Comment (saihaj) [inline src/pages/_document.tsx:70]: I have had issues in past with twitter when just using relative path. Also this is just a placeholder need a bigger image `1200 x 630`

**Comment by campbellkearns** [comment]
@saihaj could this content go inside the current UI with a paring down of some of the header nav links? are we fully abandoning the current UI?

**Comment by saihaj** [comment]
> @saihaj could this content go inside the current UI with a paring down of some of the header nav links?

Most of the things on current UI will be linking to 404s so I just thought to remove it for now. 

> are we fully abandoning the current UI?

I think there are many ideas on what we can do with this so I guess we are or we can combine things form here to that whatever we decide in our coming meetings with the team.

**Comment by campbellkearns** [comment]
@saihaj I understand where you are coming from and I agree that we will move forward to a design language/UI that _may_ look different - while it's true there may have been little to no (validated) justification to build what we built, I think that releasing a page with no design language is less optimal since this will be our "introduction" to the world, at least as far as a website goes, so I propose the following:

1. reverting to the previous UI
2. removing the overload of nav links, leaving only the links to Discord, Twitter, and FAQs
2. favor the previous design language over an empty page for aesthetics 
    - i think we can have all of the content on one page, no need for a new "route"

**Comment by saihaj** [comment]
Sure thing I can revert back to the old ui and put this information in that shell.

**Pull request closed without merging**